Sources/SwiftFusion/Image/OrientedBoundingBox.swift

```diff
@@ -39,6 +40,7 @@ public struct OrientedBoundingBox: Differentiable {
 
   /// The four corners of the region, in `(u, v)` coordinates as defined in
   /// `docs/ImageOperations.md`.
+  @differentiable
```

Comment:
Why are uFlip and vFlip called that? Is there something undocumented that needs explaining?

Reply:
@marcrasi

Reply:
I added a suggestion.